src/Commands/pm/SecurityUpdateCommands.php

```diff
@@ -84,6 +85,40 @@ public function suggestComposerCommand($updates)
         $this->logger()->notice("If that fails due to a conflict then you must update one or more root dependencies.");
     }
 
+    /**
+     * Return file_get_contents context if proxy is detected.
+     *
+     * Drupal settings override ENV ones.
+     *
+     * @return resource
+     *  A stream context resource.
+     */
+    protected function getProxySettings()
+    {
+        $settings = Settings::get('http_client_config');
+        // Check for env defined proxy then look at drupal settings one.
+        $http_proxy = getenv('http_proxy') ?: null;
```

Comment:
Could you merge it until you migrate to Guzzle ? 
Do you need any help on this migration ?
You are right about the case, since there is no standard, both cases :) should be tested (https://unix.stackexchange.com/questions/212894/whats-the-right-format-for-the-http-proxy-environment-variable-caps-or-no-ca)